### PR TITLE
Api 3633 oauth proxy remove routes const

### PR DESCRIPTION
**Pull request by bdgrider**
Remove references to  const appRoutes in index.js and replace them with config.routes.app_routes.

### CODE REVIEW

File: oauth-proxy/index.js [hunk 1]
```diff
@@ -20,18 +20,6 @@ const { configureTokenValidator } = require("./tokenValidation");
 /**
  * @deprecated - Prefer config.routes.app_routes
```
Comment (dbrown-va) [inline oauth-proxy/index.js:21]: This deprecated tag can now be removed as well.

**Review by dbrown-va — COMMENTED** [review]
Confirmed `appRoutes` removed throughout.

Confirmed auth code, refresh token, and client credentials flows, grants, revocations, introspection.  All looks good.

**Needs app config update in devops repo for redirect route.**
https://github.com/department-of-veterans-affairs/devops/blob/master/ansible/deployment/config/saml-proxy-dvp-prod.yml#L165

**Comment by bdgrider** [comment]
Config changes pull request https://github.com/department-of-veterans-affairs/devops/pull/8167

**Review by lukemarkwordtlibertyits — APPROVED** [review]
Confirmed locally

**Review by esubocz-va — APPROVED** [review]

**Pull request merged**